pymc/sampling/jax.py

```diff
@@ -515,7 +496,7 @@ def sample_jax_nuts(
     keep_untransformed: bool = False,
     chain_method: str = "parallel",
     postprocessing_backend: Literal["cpu", "gpu"] | None = None,
-    postprocessing_vectorize: Literal["vmap", "scan"] = "scan",
+    postprocessing_vectorize: Literal["vmap", "scan"] = "vmap",
```

Comment:
We should add None as an option and the default, and raise a warning if the user specifies it manually (regardless of which) so that we can safely remove the kwarg altogether in the future

Reply:
makes sense, done